rust/src/array.rs

```diff
@@ -15,6 +15,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
+use num::Zero;
```

Comment:
nit: can you move this line below `std` and above the `use array_data::{ArrayData, ArrayDataRef};`?